test/CSBondCurve.t.sol

```diff
@@ -379,4 +440,327 @@ contract CSBondCurveTest is Test {
         assertEq(bondCurve.getBondAmountByKeysCount(4, curveId), 4 ether);
         assertEq(bondCurve.getBondAmountByKeysCount(16, curveId), 10 ether);
     }
+
+    function test_viceVersa_OneInterval() public {
+        uint256[2][] memory _bondCurve = new uint256[2][](1);
+        _bondCurve[0] = [uint256(1), 0.33 ether];
```

Comment:
I'd use `UintArr(1, 0.33 ether)` but it's fine.

Reply:
Yep. It doesn't work like this :(